platform/security/sts/security-sts-attributequerycommon/pom.xml

```diff
@@ -103,7 +103,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.66</minimum>
+                                            <minimum>0.64</minimum>
```

Comment:
❓  what drove lowering the branch coverage?

Reply:
This is a result of wrapping some log statements in `if(LOGGER.isDebugEnabled())`